src/server/twemcache-rs/src/hasher.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use ahash::RandomState;
+use core::hash::BuildHasher;
+
+pub struct CacheHasher {
```

Comment:
Can we hide this within Segcache? By this I mean not exposing it outside of segcache, since Looks like it is only used in the context of `Storage<CacheHasher>`